third_party/ot-commissioner/build-android-libs.sh

```diff
@@ -4,6 +4,25 @@ readonly CUR_DIR="$(dirname "$(realpath -s "$0")")"
 
 set -e
 
+case "$TARGET" in
```

Comment:
Why?

I think this breaks the documented flow. Where does TARGET get set?

Reply:
Sorry that I didn't notice the document of how to build the commissioner has been updated. Reverted this change.